hax/treemakers/corrections.py

```diff
@@ -87,4 +168,19 @@ def extract_data(self, event):
         result['cs2_top'] = s2.area * s2.area_fraction_top * s2_top_correction
         result['cs2_bottom'] = s2.area * (1.0 - s2.area_fraction_top) * s2_bottom_correction
 
```

Comment:
My question is I can not find a specific line that defines 'cs2' in the tree maker. For eg, I see in L168 'cs2_top' is defined, can you point to me where is 'cs2'